### ISSUE TITLE
Daily average T

### ISSUE THREAD

**Issue by ribeiron** [oceanhackweek/ohw24_proj_MessageMeWhenItsHot_the_MHW_Vis-Report_app_au#6]
```[tasklist]
### Tasks
```

**Comment by ribeiron** [comment]
Some specs 
- Daily avg temperature for the two weeks around wherever date we choose in the time series. 
- X Daily avg T
- Y Time (the whole time series)
- 95th percentile line
- 5th percentile 
- Trend line
- Scatter with colours showing where each temperature falls in relation to the percentile. 

![Image](https://github.com/user-attachments/assets/f3709109-79fc-4bb0-bde0-831e5702a5b3)

**Comment by ribeiron** [comment]
This has been done. But we had a number of issues as I used the seaborn package to make the figure which made it hard to convert to HTML. In the end, I converted it to HTML, but it is non interactive. It will be good to revisit this and write it in plotly from the onset, so we can easily make this figure interactive. I will close this issue and create one of low priority to revisit this in the future.
